generator/twig/template/public/prettify.js

```diff
@@ -0,0 +1,1477 @@
+// Copyright (C) 2006 Google Inc.
```

Comment:
There is a lot of duplication here of front-end assets, maybe there is a way we can abstract?